src/adapters/configuration/adapter_config.py

```diff
@@ -364,6 +370,24 @@ class ParBnConfig(BnConfig):
     scaling: Union[float, str] = 4.0
 
 
+@dataclass(eq=False)
+class AdapterPlusConfig(BnConfig):
+    """
+    The AdapterPlus config architecture proposed by Jan-Martin O, Steitz and Stefan Roth. See https://arxiv.org/pdf/2406.06820
+    """
+
+    original_ln_after: bool = False
+    residual_before_ln: bool = True
+    stochastic_depth: float = 0.1
+    init_weights: str = "houlsby"
+    scaling: Union[float, str] = "channel"
+
+    mh_adapter: bool = False
+    output_adapter: bool = True
+    reduction_factor: Union[float, Mapping] = 2
```

Comment:
does this value correspond to the config used for experiments in the paper? I believe they use rank 8. Our reduction factor value is relative to the hidden dim of the model (768 for ViT base), ie rank = hidden_dim / reduction_factor -> reduction_factor = 768 / 8

Reply:
Yes, you are correct. At the time I wasn't sure what `reduction_factor` to use since the paper specifies they wanted a rank of 8 instead of a specified `reduction_factor`. Is it correct to assume that we should just use `reduction_factor` of 96 i.e 768/96 = 8 since the user will be using the `ViT` model with a `hidden_dim` of 768, would that be the best practice in this scenario?